jenkins/bootstrap.py

```diff
@@ -385,10 +385,17 @@ def metadata(repo, artifacts):
     if not meta or not isinstance(meta, dict):
         meta = {}
     meta['repo'] = repo
+
+    try:
+        commit = call(['git', 'rev-parse', '--short=14', 'HEAD^{commit}'], output=True)
```

Comment:
I saw two comments but github only displayed one? @rmmh did you delete the other one?